packages/labs/analyzer/src/test/browser/analyzer_test.ts

```diff
@@ -0,0 +1,39 @@
+import {assert} from '@esm-bundle/chai';
+import ts from './typescript.js';
+import {InMemoryAnalyzer} from './utils.js';
+import {AbsolutePath} from '../../index.js';
+
+suite('Analyzer', () => {
+  test('works', () => {
+    const elementPath = '/src/test-element.ts' as AbsolutePath;
+    const analyzer = new InMemoryAnalyzer({
+      typescript: ts,
+      lang: 'ts',
+      files: {
+        [elementPath]: `
+      import {LitElement, html, css} from 'lit';
```

Comment:
I expect that the declaration assertion below is a `ClassDeclaration` and not a `LitElementClassDeclaration` because stuff in the typechecker probably goes awry when it can't resolve this `lit` import? But we can't really test that you _could_ preload the cache with `lit` without coming up with a good scheme for how to do that, which I grok is out of scope of this PR for now?